### PR TITLE
[develop]: Quarterly Documentation Update (PI11)

### LINKED ISSUE #1013: Quarterly documentation update (PI11)
## Description
Periodically, documentation needs to be updated to reflect the most recent changes to the repository. Generally, updates include:
- Edits to `ConfigWorkflow.rst` to align it with `config_defaults.yaml`
- Addition of new FAQs

## Solution
Update documentation to reflect any recent changes.

### PR DESCRIPTION
**Pull request by gspetro-NOAA**
## DESCRIPTION OF CHANGES: 
Updates include:
- Edits to `ConfigWorkflow.rst` to align it with the current `config_defaults.yaml`
- Addition of new FAQs
- Improve flow of the container chapter by moving extra info to an appendix rather than including it in the main container instructions. 
- Add a `make linkcheck` function to documentation Makefile to check for any problems with links and fix issues w/links

### Type of change
- [X] This change requires a documentation update

## TESTS CONDUCTED: 
None required for documentation. The docs build successfully on my fork and can be viewed at: https://gsp-srw.readthedocs.io/en/text-us-25/

## DEPENDENCIES:
N/A

## DOCUMENTATION:
All documentation.

## ISSUE: 
Issue #1013 

## CHECKLIST
- [X] My code follows the style guidelines in the Contributor's Guide
- [X] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas - N/A
- [X] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain). - N/A
- [X] My changes generate no new warnings
- [ ] New and existing tests pass with my changes - N/A no testing required
- [X] Any dependent changes have been merged and published N/A - no dependent changes

## CONTRIBUTORS: 
@maddenp-noaa

### CODE REVIEW

File: docs/UsersGuide/source/Reference/FAQ.rst
Comment (gspetro-NOAA) [inline docs/UsersGuide/source/Reference/FAQ.rst:10]: FYI this will render as a hyperlinked list of contents populated based on the page structure, similar to what we had before, but not manual.

**Comment by gspetro-NOAA** [comment]
@MichaelLueken Currently, the `make linkcheck` command highlights 2 "broken" links, both from Intel. These links work in the browser, and a `curl -I https://link...` brings up a 200. Since this PR is not introducing any new errors, I was hoping we could proceed with review anyway while I explore solutions for ignoring or removing those errors.

**Comment by MichaelLueken** [comment]
> @MichaelLueken Currently, the `make linkcheck` command highlights 2 "broken" links, both from Intel. These links work in the browser, and a `curl -I https://link...` brings up a 200. Since this PR is not introducing any new errors, I was hoping we could proceed with review anyway while I explore solutions for ignoring or removing those errors.

@gspetro-NOAA - Sure, we can go ahead and proceed with reviewing the current documentation changes while you look into the `make linkcheck` issues.  Once the necessary approvals are received, I will hold off on merging until you give the go ahead.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (RatkoVasic-NOAA) [inline docs/UsersGuide/source/BackgroundInfo/Introduction.rst:63]: Suggested: Level 1 platforms

**Review by RatkoVasic-NOAA — APPROVED** [review]
Looks good to me. Links work.

**Review by MichaelLueken — APPROVED** [review]
@gspetro-NOAA - These changes look good!  I was also able to test the links, which work without issue.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst:63]: I think it's probably fine as-is because a user might have access to one platform or more than one. Grammatically, "access to a Level 1 platform" or "access to Level 1 platforms" are both correct. :)

**Comment by gspetro-NOAA** [comment]
@MichaelLueken I found a solution for the link issue using `linkcheck_ignore` in the `conf.py` file, so now `make linkcheck` should return no errors! :)

**Comment by MichaelLueken** [comment]
> @MichaelLueken I found a solution for the link issue using `linkcheck_ignore` in the `conf.py` file, so now `make linkcheck` should return no errors! :)

@gspetro-NOAA - Nice!  I'll go ahead and get this PR merged in the morning, then we can work on PR #1018.

**Pull request merged**
